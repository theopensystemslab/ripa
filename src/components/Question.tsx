import { makeStyles } from "@material-ui/core/styles";
import Typography from "@material-ui/core/Typography";
import React from "react";

const useStyles = makeStyles(theme => ({
  question: {
    width: "100%",
    padding: 0,
    zIndex: 10,
    position: "relative"
  }
}));

const Question = ({ children, ...props }) => {
  const classes = useStyles();
  return (
    <Typography
      className={classes.question}
      variant="h4"
      component="div"
      {...props}
    >
      <strong>{children}</strong>
    </Typography>
  );
};

export default Question;
