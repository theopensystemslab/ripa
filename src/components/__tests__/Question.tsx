import { cleanup, render } from "@testing-library/react";
import * as React from "react";

import Question from "../Question";

afterEach(cleanup);

describe("Question Component", () => {
  it("render snapshot", () => {
    const children = "This is a question";
    const { asFragment } = render(<Question>{children}</Question>);
    expect(asFragment()).toMatchSnapshot();
  });
});
