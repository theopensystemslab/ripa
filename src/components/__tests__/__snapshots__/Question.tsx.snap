// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question Component render snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiTypography-root makeStyles-question-1 MuiTypography-h4"
  >
    <strong>
      This is a question
    </strong>
  </div>
</DocumentFragment>
`;
